src/components/Pages/Trending/Trending.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import CustomPagination from "../../Pagination/CustomPagination";
+import SingleContent from "../../SingleContent/SingleContent";
+import "./Trending.css";
+
+const Trending = () => {
+  const [page, setPage] = useState(1);
+  const [content, setContent] = useState([]);
+
+  const fetchTrending = async () => {
+    const { data } = await axios.get(
```

Comment:
How did you discover axios? I'm just courious why you decided to use `axios` instead of `fetch`

Reply:
I have changed the function from axis to fetch.